components/camel-slack/src/main/java/org/apache/camel/component/slack/SlackProducer.java

```diff
@@ -59,7 +61,7 @@ protected void doStop() throws Exception {
     }
 
```

Comment:
And you have a doStop method where you stop/close the client?

Reply:
I added that in previous commit (https://github.com/apache/camel/pull/4990/commits/424273684ce44708461b0188cc90832517aae26b)